presto-native-execution/presto_cpp/main/TaskResource.cpp

```diff
@@ -342,7 +342,23 @@ proxygen::RequestHandler* TaskResource::deleteTask(
         }
 
         json taskInfoJson = *taskInfo;
-        http::sendOkResponse(downstream, taskInfoJson);
+        std::string messageBody;
+        try {
+          messageBody = taskInfoJson.dump();
```

Comment:
Don't we have other places like this? Should we handle all of them simiarily?

Reply:
@mbasmanova 
Ideally yet, do you want me to cover all of them in this change?

Reply:
That would be nice. Maybe add a helper function to avoid copy-pasting too much.